marlowe-dashboard-client/src/Marlowe/README.md

```diff
@@ -0,0 +1,38 @@
+# About Marlowe Contracts and the PAB
+
+"Contract" in the context of this app can mean several things:
+
+1. A Marlowe contract (a program that conforms to the schema defined in Marlowe.Semantics).
+2. An instance of a Marlowe contract installed on the blockchain. These can be identified
+   by their `MarloweParams`.
+3. A Plutus contract. Marlowe Run uses three Plutus contracts to operated (see below).
+4. An instance of a Plutus contract running in the PAB. These are identified by a
+   `ContractInstanceId`.
```

Comment:
I looked at this after I made the comments in `MarlowParams` 😅, but maybe we can:

* rename `ContractInstanceId` to `PlutusInstanceId` or `PlutusContractInstanceId` or `PlutusContractId`
* rename `MarloweParams` to `MarloweInstanceId`. 
 
And from some other comments I read in this PR, if we really don't need to access the stuff inside `MarloweParams`/`MarloweInstanceId` we can avoid exporting it's constructor and treat it as an opaque type.

Reply:
Again, I'm just following the Haskell names for now. We can revisit later though.